src/main/java/org/dita/dost/util/Constants.java

```diff
@@ -941,7 +945,7 @@
     public static final String ATTRIBUTE_NAME_NAME = "name";
     /**type attribute value subjectScheme.*/
     public static final String ATTR_TYPE_VALUE_SUBJECT_SCHEME = "subjectScheme";
-    /**store how many scheme files a ditamap file used in form of {@code Map&lt;String, Set&lt;String>>}.*/
+    /**store how many scheme files a ditamap file used in form of Map<String, Set<String>>.*/
```

Comment:
Not sure what this was - thought I was *undoing* a change that had been introduced with the Eclipse auto-formatting of Java.